legal-api/src/legal_api/models/dc_revocation_reason.py

```diff
@@ -0,0 +1,31 @@
+# Copyright © 2023 Province of British Columbia
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""This module holds revocation reasons for issued credential."""
+from enum import Enum
+
+
+class DCRevocationReason(Enum):
+    """Digital Credential Revocation Reasons."""
+
+    UPDATED_INFORMATION = 'You were offered a new credential with updated information ' \
+        'and that revoked all previous copies.'
+    VOLUNTARY_DISSOLUTION = 'You chose to dissolve your business. ' \
+        'A new credential was offered that reflects the new company status and that revoked all previous copies.'
+    ADMINISTRATIVE_DISSOLUTION = 'Your business was dissolved by the Registrar.'
+    PUT_BACK_ON = 'Your business was put back on the Registry. '
+    RESTORATION = 'Your business was put back on the Registry. ' \
```

Comment:
Need different text here. Check with Mihai or Yui or Janis. I'm thinking it will be something like, "Your business was restored."

Note that some restorations are limited, meaning they expire (unless extended or converted to full). We will eventually have a batch job to dissolve limited restorations. Will that dissolution be caught by this code?